src/plugin.js

```diff
@@ -51,6 +51,19 @@ function isTag(path, tagName, allowGlobal = false) {
   );
 }
 
+function isStyled(path, styledTag) {
+  if (styledTag) {
+    return (
+      t.isCallExpression(path.node.tag) &&
+      path.get('tag.callee.name').node === styledTag
+    );
+  }
+  return (
+    t.isCallExpression(path.node.tag) &&
+    path.get('tag.callee').referencesImport('css-literal-loader/styled')
```

Comment:
or am i misunderstanding? test coverage here might be nice

Reply:
I do not cover `import { styled as foo } from 'css-literal-loader/styled'` case, to make PR atomic as possible

This option should be used if want to miss `import`. In this case you set `styledTag` to `"styled"`.

If you want to change origin logic to cover `{ styled as foo }` I can do it, but it could be potentially breaking change.

Reply:
In other words, if you missed `styledTag` we will use old behavior. If you will set it, you can miss the import.

Reply:
> test coverage here might be nice

https://github.com/4Catalyzer/css-literal-loader/pull/36/files#diff-5086c2c2ab0988d46c360a8497a51fe0